src/BuildScriptGeneratorCli/Commands/InstallRunTimeCommand.cs

```diff
@@ -0,0 +1,71 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+using McMaster.Extensions.CommandLineUtils;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Oryx.BuildScriptGenerator;
+using Microsoft.Oryx.Common;
+
+namespace Microsoft.Oryx.BuildScriptGeneratorCli
+{
+    [Command(InstallRunTimeCommand.Name, Description = "Install the required runtime components for a platform.")]
```

Comment:
For my own understanding, what is this command meant to do? an E2E scenario would be helpful

Reply:
It'll install the actual platform - but just the runtime components (not SDKs)

Reply:
So this command generates a bash script which installs the runtime components? say for example if .NET Cor runtime 2.1 needs to be installed, does the generated bash script install it by downloading the content?

Reply:
Correct